ui/lib/core/addon/components/search-select-with-modal.js

```diff
@@ -2,8 +2,7 @@ import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
-import { singularize } from 'ember-inflector';
-import { resolve } from 'rsvp';
+import { task } from 'ember-concurrency';
```

Comment:
most of these changes are taken from the now glimmerized search select component